Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -238,6 +258,18 @@ end CircleIntegrable
 theorem circleIntegrable_zero_radius {f : ℂ → E} {c : ℂ} : CircleIntegrable f c 0 := by
   simp [CircleIntegrable]
 
+/-- Circle integrability is invariant when functions change along discrete sets. -/
+theorem circleIntegrable_congr_codiscreteWithin {c : ℂ} {R : ℝ} {f₁ f₂ : ℂ → ℂ}
```

Comment:
This one should rather be named `CircleIntegrable.congr_of_codiscreteWithin`, and then you should have another one named `circleIntegrable_congr_of_codiscreteWithin` which would be the iff version.